Light Vox Engine/Engine.cpp

```diff
@@ -107,6 +111,7 @@ HRESULT Engine::Run()
 		{
 			graphics->Update();
 			graphics->Render();
+            time->UpdateTimer();
```

Comment:
Oh no I think this was a remnant from the tab and spaces thing -- I'll put an issue to fix this between releases